### PR TITLE
Fixing more typos.

### PR DESCRIPTION
**Pull request by semercim**
*Değişikliklerin açıklaması:*

Bu çekme talebini göndererek, bu katkıyı tercih ettiğiniz şartlar 
altında kullanabileceğinizi, değiştirebileceğinizi, kopyalayabileceğinizi 
ve yeniden dağıtabileceğinizi onaylıyorum.

### CODE REVIEW

**Comment by semercim** [comment]
Hello @AnirudhDagar , I want to merge this typo fix branch. But CI/CD fails in bahdanau-attention.md. Could you, kindly, help fix it?

**Comment by AnirudhDagar** [comment]
Hi Murat, sure I'll take a look and try to fix this today.

File: Jenkinsfile [hunk 1]
```diff
@@ -38,6 +38,7 @@ stage("Build and Publish") {
       conda activate ${ENV_NAME}
       ./static/cache.sh restore _build/eval_pytorch/data
       d2lbook build eval --tab pytorch
+      d2lbook build slides --tab pytorch
```
Comment (AnirudhDagar) [inline Jenkinsfile:41]: Why do we need this? The slides feature was specific to d2l-en, and I don't think we have any separate slides repos specifically for different translations at the moment.
```suggestion
```

File: static/build.yml [hunk 2]
```diff
@@ -5,9 +5,9 @@ dependencies:
...
-    - torch==1.11.0+cu102
+    - torch==1.12.0+cu102
     - -f https://download.pytorch.org/whl/torch_stable.html
-    - torchvision==0.12.0+cu102
+    - torchvision==0.13.0+cu102
     - -f https://download.pytorch.org/whl/torch_stable.html
-    - tensorflow==2.8.0
-    - tensorflow-probability==0.16.0
+    - tensorflow==2.9.1
+    - tensorflow-probability==0.17.0
```
Comment (AnirudhDagar) [inline static/build.yml:13]: Since we are updating libraries here, I'd suggest let's rerun the CI from scratch here for the whole project before merging.

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
I've made a couple of comments which are orthogonal to the CI issue. Recently it seems that a lot of CI jobs have been timing out. I don't have a good answer for why that is happening, but I'll just ask d2l-bot to rebuild this whole PR from scratch to address my other comment. If everything goes well, we should be green.

**Comment by AnirudhDagar** [comment]
@d2l-bot please rebuild :)

**Comment by semercim** [comment]
@d2l-bot please rebuild

File: static/build.yml (hunk 2 above)
Comment (semercim) [reply static/build.yml:13]: OK. Makes sense. But it failed again in the rebuild at another notebook creation and it looks like I do not have permissions to activate d2l-bot actions. @AnirudhDagar, could you, kindly, either rebuild or give me more power to rule d2l-bot :D ?

**Comment by AnirudhDagar** [comment]
@d2l-bot should work for everyone and that is why we added it. I think I need to configure it for the repo particularly. It is not being triggered by GitHub API.

Besides the CI seems to have failed due to a different issue this time. Maybe that also needs fixing.

**Comment by AnirudhDagar** [comment]
@d2l-bot please rebuild :)

**Comment by semercim** [comment]
> 

@AnirudhDagar  It looks like now that it fails at some different notebook at each build. Previously it was failing only the tensorflow version and now it also fails the mxnet version :/

**Comment by AnirudhDagar** [comment]
@semercim multiple flaky fails can happen when we run 175 notebooks across three frameworks. A few which are common and hard to pin-point fix include:
1. CI Timeouts: can sometimes happen because of resource availability lock. (happened [here](http://ci.d2l.ai/blue/organizations/jenkins/d2l-en/detail/PR-2213/6/pipeline)) 
2. Datasets/pretrained-models: can have issues with download, leading to weird runtime bugs. (happened [here](http://ci.d2l.ai/blue/organizations/jenkins/d2l-tr/detail/PR-63/25/pipeline))

I've restarted the PR build again and everything should be green now.

**Comment by semercim** [comment]
Thanks for your help @AnirudhDagar . Everything seems OK.

**Pull request merged**